### PR TITLE
Adding test apps to build

### PR DESCRIPTION
**Pull request by jasonsandlin**

### CODE REVIEW

File: build/GDK/TestClientApp.vcxproj [hunk 1]
```diff
@@ -22,6 +22,7 @@
     <!-- - - - -->
     <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
     <TargetRuntime>Native</TargetRuntime>
+    <XsapiBuildFromSource>true</XsapiBuildFromSource>
```
Comment (jplafonta) [inline build/GDK/TestClientApp.vcxproj:25]: ><XsapiBuildFromSource>true</XsapiBuildFromSource> [](http://example.com/codeflow?start=4&length=49)

remind me what this controls? Does it need to be "XsapiBuildFromSource"?

Comment (jasonsandlin) [reply build/GDK/TestClientApp.vcxproj:25]: without we get a ton of errors like 

##[error]libHttpClient.141.GDK.C.lib(global_publics.obj)(0,0): Error LNK2005: HCInitialize already defined in libHttpClient.141.GDK.C.lib(global_publics.obj)

cause we're building libhttpclient from source in this project.  I'll chat with you about changes we should make to gdk props outside this PR around this

Comment (jplafonta) [reply build/GDK/TestClientApp.vcxproj:25]: Another approach here would be to explicitly remove the XSAPI dependency for this project.  This should probably be done anyhow, and then we wouldn't need this weird XSAPI related property in a PlayFab project.

**Review by jplafonta — APPROVED** [review]
:shipit:

**Pull request merged**